lib/nexpose/site.rb

```diff
@@ -488,6 +487,10 @@ def to_h
           organization: @organization.to_h,
           users: users
       }
+      # Only pass in blackouts if they were actually specified
+      hash[:blackouts] = Array(@blackouts).map(&:to_h) if @blackouts && @blackouts.any?
```

Comment:
Wrapping this in `Array` isn't buying anything since the `if`modifier will not accept any null/empty values.